src/main/java/com/exadel/discount/config/WebMvcConfig.java

```diff
@@ -0,0 +1,18 @@
+package com.exadel.discount.config;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.HandlerInterceptor;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+@RequiredArgsConstructor
+public class WebMvcConfig implements WebMvcConfigurer {
+    private final HandlerInterceptor handlerInterceptor;
```

Comment:
Probably you want to inject your LoggerInterceptor and in application it could be several interceptors, so inject with Qualifier here

Reply:
+